bears/jinja2/Jinja2Bear.py

```diff
@@ -123,6 +181,47 @@ class Jinja2Bear(LocalBear):
     CONTROL_END_REGEX = re.compile(
         r'(?P<open>{%[+-]?)(?P<content>\s*end(for|if)\s*?)'
         r'(?P<close>[+-]?%})(?P<label>{#.*?#})?')
+    LABEL_REGEX = re.compile(
+        r'(?P<open>{%[+-]?)'
+        r'(?P<label>\s*(end)*(for|if|block|set|elif|else))')
```

Comment:
put `for|if|block|set|elif|else` into a list, as a module level constant.  then explode it here using `'|'.join(BLAH_KEYWORD_LIST)`